README.md

```diff
@@ -1,55 +1,61 @@
 
 Live tests: [![Live Tests](https://semaphoreci.com/api/v1/flowcrypt/flowcrypt-browser/branches/master/badge.svg)](https://semaphoreci.com/flowcrypt/flowcrypt-browser)
 
-This project is written in TypeScript. Browsers only understand JavaScript, so the project needs to be transpiled. You need to build the project the first time you download it, and build it after each change you make to see the result in the browser. To get started, please follow the instructions below:
+This project has been written in TypeScript. However, since browsers only understand JavaScript, the project needs to be transpiled to it. You need to build the project the first time you download/clone it, and then build it again after each change you make to see the result in the browser. To get started, please follow the instructions below:
+
+1. Navigate to the project folder and install the tooling by running the following commands:
 
-1. Install the tooling by running the following command after navigating to the appropriate folder:
 ```bash
+# Navigate to the appropriate folder
 $ cd some/folder/flowcrypt-browser
+# Install the tooling
 $ npm install
 ```
+
 2. To build the project (currently available for Linux and Mac only), run the following command:
 
 ```bash
 $ npm run build
 ```
-After running this command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+After executing the build command, you can find your built project in the `build/chrome-consumer` and `build/firefox-consumer` folders.
+
+> Note: The `build` folder also contains other versions of the browser extension.
 
 To load the extension in Google Chrome, please follow these steps:
 
 1. Open your Chrome browser and navigate to `chrome://extensions/`.
-2. If not already enabled, toggle on the `Developer mode`    switch located at the top-right corner.
-3. Click on the `Load Unpacked` button.
-4. Browse to and select the appropriate folder, either `build/chrome-consumer` or `chrome-enterprise`.
+2. If it isn't already enabled, toggle on the **Developer mode**. The switch button is located at the top-right corner.
+3. Click on the `Load unpacked` button.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/chrome-consumer` or `chrome-enterprise`.
 
-Similarly, to load the extension in Firefox, follow these steps:
+Similarly, to load the extension in Firefox, please follow these steps:
 
 1. Open your Firefox browser and navigate to `about:debugging`.
 2. Click on the `This Firefox` tab.
 3. Click on the `Load Temporary Add-on` button located at the top-right corner.
-4. Browse to and select the appropriate folder, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
+4. Browse to the `flowcrypt-browser/build` folder and select the appropriate project version, either `build/firefox-consumer` or `firefox-enterprise`, and select the `manifest.json` file.
```

Comment:
currently we build only `firefox-consumer` version, `firefox-enterprise` can be removed from this line